config/l5-swagger.php

```diff
@@ -288,7 +288,7 @@
          */
         'constants'             => [
             'L5_SWAGGER_CONST_HOST' => env('L5_SWAGGER_CONST_HOST', 'http://localhost:8000'),
-            'L5_SWAGGER_CONST_ENDPOINT' => env('L5_SWAGGER_CONST_HOST', 'http://localhost:8000') . '/api/v1',
```

Comment:
This needs to be changed back because you won't be able to test the api locally via swagger.

<img width="835" alt="Screenshot 2024-07-02 at 12 39 26" src="https://github.com/Traewelling/traewelling/assets/1267894/83b18355-efe5-4c1d-a5e1-db107ce1ed23">